app/models/spotlight/main_navigation.rb

```diff
@@ -23,5 +27,13 @@ def label_or_default
     def default_label
       I18n.t(:"spotlight.main_navigation.#{nav_type}")
     end
+
+    private
+
+    ##
+    # Allows us to scope translations namespace.
+    def slug
+      ['main_navigation', nav_type].join('.')
```

Comment:
While this `main_navigation` isn't absolutely necessary, I thought it might help give context to someone who came across the key.